packages/rum-core/test/createDomMutationStub.ts

```diff
@@ -0,0 +1,25 @@
+import { DOMMutation } from '../src/browser/domMutationObservable'
+
+export interface DOMMutationStub extends DOMMutation {
+  notify: () => void
+}
+
+export function createDomMutationStub(): DOMMutationStub {
+  let callbacks: Array<() => void> = []
+
+  function notify() {
+    callbacks.forEach((callback) => callback())
+  }
+
+  return {
+    notify,
+    subscribe: (callback: () => void) => {
+      callbacks.push(callback)
+      return {
+        unsubscribe: () => {
+          callbacks = callbacks.filter((other) => callback !== other)
+        },
+      }
+    },
+  }
+}
```

Comment:
this seems to match the `Observable` API, maybe we don't need it and we could use an `Observable` directly to stub it?

Reply:
Now that I've implemented the `unsubscribe()` in `Observable`, it seems reasonable.